src/command_modules/azure-cli-monitor/setup.py

```diff
@@ -30,7 +30,9 @@
 DEPENDENCIES = [
     'azure-cli-core',
     'azure-monitor==0.3.0',
-    'azure-mgmt-monitor==0.2.1'
+    'azure-mgmt-monitor==0.3.0',
+    # NOTE: make sure the version of the managed resource package is aligned with VM and other command modules.
```

Comment:
What is this comment for?

Reply:
There is another PR is changing this dependency's version in the other command modules, but it is not merged yey. I'd like to remind myself to ensure the dependencies are aligned by the time I merge this PR.